traffic_ops/v4-client/server_update_status.go

```diff
@@ -52,10 +53,14 @@ func (to *Session) SetServerQueueUpdate(serverID int, queueUpdate bool, opts Req
 // SetUpdateServerStatuses updates a server's queue status and/or reval status.
 // Either updateStatus or revalStatus may be nil, in which case that status
 // isn't updated (but not both, because that wouldn't do anything).
+//
+// Deprecated: Prefer to use SetUpdateServerStatusTimes
```

Comment:
Same as above RE: Godoc

Reply:
This should just be removed from the v4 client, it's already marked deprecated in v3. There's no reason to wait until v5 to get rid of this.

Reply:
You're right, but I'm actually not deprecating this in v3. It is now deprecated in V4. We need to maintain both possibilities to ensure currently deployed T3C clients continue to function with new TO development since we can't guarantee simultaneous deployment of both components.